src/app/contexts/RequestContext/getOriginContext/index.test.ts

```diff
@@ -17,27 +17,27 @@ describe('getOriginContext', () => {
     {
       bbcOrigin: undefined,
       location: undefined,
-      expected: { isUK: true, origin: 'https://www.bbc.co.uk' },
+      expected: { origin: 'https://www.bbc.co.uk' },
       assertion: 'should return defaults if no origin can be found',
     },
     {
       bbcOrigin: 'https://foobar.com',
       location: 'https://beepboop.co.uk',
-      expected: { isUK: false, origin: 'https://foobar.com' },
+      expected: { origin: 'https://foobar.com' },
```

Comment:
Can we look at these tests again to see if they are all needed. Many of them refer to the orgin determining `isUK` when `isUK` is no longer involved in the `OriginContext`.